### DISCUSSION TITLE
What if accuracy is high but loss is not that good for text detection training?

### DISCUSSION THREAD

**Discussion by light42** [PaddlePaddle/PaddleOCR#8033] [Q&A] [answered]
I'm training text detection with ResNet_v50 model, and dataset synthetically generated from [this repo](https://github.com/WenmuZhou/TableGeneration/tree/main/TableGeneration).
I am using default [configs](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/det/det_r50_db%2B%2B_icdar15.yml), with slight adjusment at learning_rate(0.0006) and weight_decay(0.0001).

It achieved very high accuracy (~0.99) yet the loss has been so-so (dropped from ~6.00 to ~1.00).
My question is, is this already good enough? And if it is not good, what can I do to improved it further?

**Comment by LDOUBLEV** [comment, marked as answer]
Loss is not an indicator for evaluating the quality of the model, acc is.

The calculation formula of loss is different, and the output value will be stable within a certain range, not necessarily a small value. 

BTW, the loss of GAN will even have negative values.
